gateway/test/network/network.routes.test.ts

```diff
@@ -65,3 +65,36 @@ describe('GET /network/config', () => {
       .expect(200);
   });
 });
+
+describe('GET /network/tokens', () => {
+  it('should return 200 when retrieving tokens', async () => {
+    await request(gatewayApp)
+      .get(`/network/tokens`)
+      .query({
+        chain: 'ethereum',
+        network: 'kovan',
+      })
+      .expect('Content-Type', /json/)
+      .expect(200);
+  });
+  it('should return 200 when retrieving specific tokens', async () => {
+    await request(gatewayApp)
+      .get(`/network/tokens`)
+      .query({
+        chain: 'ethereum',
+        network: 'kovan',
+        tokenSymbols: ['COIN3', 'COIN1'],
+      })
+      .expect('Content-Type', /json/)
+      .expect(200);
+  });
+  it('should return 500 when retrieving tokens for invalid chain', async () => {
+    await request(gatewayApp)
+      .get(`/network/tokens`)
+      .query({
+        chain: 'unknown',
+        network: 'kovan',
+      })
+      .expect(500);
+  });
+});
```

Comment:
During one of my conversations with James, I think this is in fact calling a request to the actual endpoint and not mocking the request. Is it possible that we use `patch` and `unpatch` to mock these requests?

Reply:
There's no point patching in this case. The tokens are located in the project. It's not making an external http call in this case.